uber/config.py

```diff
@@ -303,6 +303,28 @@ def REMAINING_BADGES(self):
     def ADMIN_ACCESS_SET(self):
         return sa.AdminAccount.access_set()
 
+    @request_cached_property
+    def EMAIL_APPROVED_IDENTS(self):
+        return self.get_approved_idents()
+
+    # TODO: only reason this is a separate method is because for unit tests we patch this method,
+    # and monkeypatch.setattr() doesn't like to patch EMAIL_APPROVED_IDENTS
+    # when it's decorated with @request_cached_property
+    def get_approved_idents(self):
+        with sa.Session() as session:
+            return {ae.ident for ae in session.query(sa.ApprovedEmail)}
```

Comment:
Ah, gotcha. That totally makes sense. I agree that for now, having this PR finished is better than having perfect form regarding where this is kept. :)